python/smashbox/utilities/__init__.py

```diff
@@ -37,17 +37,19 @@ def reset_owncloud_account(reset_procedure=None, num_test_users=None):
         logger.info('reset_owncloud_account (%s) for %d users', reset_procedure, num_test_users)
 
     if reset_procedure == 'delete':
```

Comment:
I think it needs some addition to smashbox. Some cheching if oc_admin_password was set.